README.md

```diff
@@ -1,2 +1,24 @@
 # hw01
 Repository for homework 1
+
+## Eigen Aoki
```

Comment:
Evaluation HW-01 for Eigen Aoki
----------------------------------------------------
| Overall                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    x       |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |     x      |              |            |

Remarks:

-Great overall work on the assignment! From looking at your code I learned that you can number lists without just using 1. over and over again, and that you can put in links directly with their full url instead of having to use parentheses 
-From reading your pull request comment I also learned you can clone a repository through R, which seems easier than using the terminal as I was doing. It also seems easier to use it to push a file as well and gives you more control than having to push all the files in a folder as I confusedly did through the terminal. 
-Suggestions for improving subheading and presentation of links is given in other comments

Reply:
Actually, might be useful to make sure you can do everything from the terminal for Git in case you end up coding later down the line outside of an IDE.